srml/pow/src/diffs/average_span.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018-2019 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+use srml_support::{StorageValue, decl_storage, decl_module};
+use srml_support::traits::Get;
+use pow_primitives::Difficulty;
+use sr_primitives::traits::{UniqueSaturatedInto, Zero};
+
+pub trait Trait: timestamp::Trait {
+	type Span: Get<Self::BlockNumber>;
+	type TargetPeriod: Get<Self::Moment>;
+	type InitialDifficulty: Get<Difficulty>;
+}
+
+decl_storage! {
+	trait Store for Module<T: Trait> as AverageSpanDifficultyAdjustment {
+		LastTimestamp get(last_timestamp): Option<T::Moment>;
```

Comment:
I guess this is more the timestamp of the last span? If so, can we either leave a comment or rename appropriately?